build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -190,6 +190,28 @@ def _convert_target_list_block(list_name, targets):
   return _convert_string_list_block(list_name, targets, sort=True, quote=False)
 
 
+# Copied from integrations/tensorflow/e2e/iree_e2e_cartesian_product_test_suite.bzl
```

Comment:
I don't think this comment is useful to keep in bazel_to_cmake. The function stands on its own without that context (which isn't sufficient on its own to explain why it exists, and will be outdated if that file is moved/removed).

Reply:
Well except that the function has Starlark weirdness, as Stella commented on :-D. If the file is moved/removed, then someone should update references to it and it will still be in the git history.

Reply:
Yeah, just trying to keep the conversion as generic as possible. Something referencing tensorflow integration tests is particularly not generic :P

Reply:
The whole function implemented below is also defined in `integrations/tensorflow/e2e/iree_e2e_cartesian_product_test_suite.bzl` though...